### ISSUE TITLE
OpenRace Crashes on several Data Race Benchmarks

### ISSUE THREAD

**Issue by markbcahill** [coderrect-inc/OpenRace#211]
On a fresh pull of the OpenRace Docker container, OpenRace crashes when run on several .ll files from the provided /dataracebench/ll folder. The specific files are:

003, 021, 027, 031, 037, 046, 053, 057, 058, 060-065, 072-074, 076, 078, 079, 084, 085, 091-097, 100-103, 105-107, 109, 110, 113, 114, 116, 117, 121-123, 127-136, 139-158, 160-164, 169, 170, 172

**Comment by BradSwain** [comment]
Thanks for taking a look at our docker container.

We don't fully support all DRB cases yet, so crashing on some is expected. 
However, on the current develop branch the large majority of these are supported.

The docker image is not updated regularly, so the version of openrace on that image is from ~1 month ago before we added support for many OpenMP features.


I've opened an issue to have the docker image updated from the develop branch nightly (see #212), so that it is at least somewhat up to date.

However, until we make an official release, the best way to try the best and most up to date version of tool is tobuild it from source following this guide: https://coderrect-inc.github.io/OpenRace/doc-setup-dev-env/
